components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.common/src/main/java/org/wso2/carbon/identity/api/server/identity/governance/common/GovernanceConstants.java

```diff
@@ -65,7 +65,9 @@
                 "Unable to get the identity governance preferences.",
                 "Server Encountered an error while retrieving identity governance connector preferences."),
         ERROR_CODE_INCORRECT_CONNECTOR_NAME("50011", "Invalid connector name",
-                "Unable to find the connector with the name %s.");
+                "Unable to find the connector with the name %s."),
```

Comment:
Unable to find the connector with the name %s. -> Unable to find a connector with the name %s.